django/core/management/commands/loaddata.py

```diff
@@ -222,57 +243,67 @@ def load_label(self, fixture_label):
                     RuntimeWarning
                 )
 
+    def _get_fixture_name_and_dirs(self, fixture_name):
+        dirname, basename = os.path.split(fixture_name)
+        if os.path.isabs(fixture_name):
+            fixture_dirs = [dirname]
+        else:
+            fixture_dirs = self.fixture_dirs
+            if os.path.sep in os.path.normpath(fixture_name):
+                fixture_dirs = [os.path.join(dir_, dirname) for dir_ in fixture_dirs]
+        return basename, fixture_dirs
+
+    def _get_targets(self, fixture_name, ser_fmt, cmp_fmt):
+        databases = [self.using, None]
+        cmp_fmts = list(self.compression_formats) if cmp_fmt is None else [cmp_fmt]
```

Comment:
`product()` handles iterables so we don't need to call `list()`:
```suggestion
        cmp_fmts = self.compression_formats if cmp_fmt is None else [cmp_fmt]
```

Reply:
Agreed. I hadn't done this originally because I just moved the code without editing so that it'd show up as moved in `git diff --color-moved`.